### PR TITLE
Implement Fixed Position Navigation Box 

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR is a response to feedback I received from a colleague regarding the accessibility of the feedback as you scroll down the page. In an effort to make the navigation accessible throughout the page, I have added a fixed combo box that stays on the view port. 
image

_Snap shot of Navigation screen with combobox in the corner_

- **Create and style fixed nav combobox**
- **Connect combobox change event to navigate function**

_Potential stretch feature: Have combobox value change on scroll, in line with the current slide in view_

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -31,7 +31,12 @@ const initialiseNavigation = () => {
     item.addEventListener('click', function() {
       navigate(this.getAttribute('data-link'));
     })
-  }
+  };
+  
+  const sectionSelector = document.getElementById("section-selector");
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/script.js]: nit: use single quotes, here and below

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -31,7 +31,12 @@ const initialiseNavigation = () => {
...
+  sectionSelector.addEventListener("change", function(e) {
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/script.js]: nit: add a space between `function` and `(`

**Review by krasimirgg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/script.js]: Does this change also apply to functions without arguements? e.g. function()

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/script.js]: Updated with commit f4a5c10

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/script.js]: * Updated here for commit f4a5c10 but question pending above

**Review by krasimirgg — APPROVED** [review]
Nice to get the navbox in!

**Pull request merged**